WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift

```diff
@@ -167,25 +165,26 @@ class AppSettingsViewController: UITableViewController {
 
     override func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
         if shouldShowEditorFooterForSection(section) {
-            let view = tableView.dequeueReusableHeaderFooterView(withIdentifier: AppSettingsEditorFooterView.reuseIdentifier) as! AppSettingsEditorFooterView
-
-            view.tapBlock = { [weak self] in
-                WPAppAnalytics.track(.editorAztecBetaLink)
-
-                if let controller = self {
-                    WPWebViewController.presentWhatsNewWebView(from: controller)
-                }
-            }
-
-            return view
+            let footer = UITableViewHeaderFooterView.init(frame: CGRect(x: 0.0,
```

Comment:
You can lose the `.init` here and just have `UITableViewHeaderFooterView(frame: ...)`